ggshield/core/scan/file.py

```diff
@@ -81,9 +85,9 @@ def get_files_from_paths(
             " Use --recursive to scan directories."
         )
 
-    files = list(generate_files_from_paths(filepaths, verbose))
+    files = list(generate_files_from_paths(filepaths, display_binary_files))
```

Comment:
Shouldn't we rename the argument in the function also? (since it won't reflect the value of `verbose` in all cases)